config/rbac/percona_mongodb_clusterrole.yaml

```diff
@@ -12,4 +12,5 @@ rules:
       - perconaservermongodbs/status
     verbs:
       - get
+      - watch
```

Comment:
Why do you change the clusterrole here, is that on purpose?

If it is not intentional or required it should be removed from the PR.

Reply:
I just copied the yaml from the downstream.. that one has captured the watch verb in the list as per @baijum's suggestion, but the upstream doc was missing this and I had to update it. So yes, it is intentional.